cmd/vic-machine/configure/configure.go

```diff
@@ -217,6 +238,39 @@ func updateSessionEnv(sess *executor.SessionConfig, envName, envValue string) {
 	sess.Cmd.Env = newEnvs
 }
 
+func (c *Configure) processCertificates(conf *config.VirtualContainerHostConfigSpec) error {
```

Comment:
This is for processing networking stuff, which we discussed on Slack :)  I could pass in the `Networks` struct instead and do the data copying outside of this method, but I'm not sure that would be cleaner.